### ISSUE TITLE
Preview link in passenger showcase guide

### ISSUE THREAD

**Issue by DenisovichDev** [CodingTrain/thecodingtrain.com#702]
Under [SUBMITTING A PULL REQUEST](https://thecodingtrain.com/guides/passenger-showcase-guide#submitting-a-pull-request) I think it's worth mentioning that in the description of the PR it's good to have a preview link. Since we don't have that template thing like we did in the old repo anymore.

I think this screenshot could have a preview link or picture:
![image](https://user-images.githubusercontent.com/66998096/197183914-19ba8f63-7fdd-4404-baaa-d8008e142c0c.png)

**Comment by shiffman** [comment]
Good call! I'm also not opposed to adding templates for issues and pull requests if you have ideas on what would be most helpful to the community!

**Comment by DenisovichDev** [comment]
@shiffman I can work on some templates for PR and issues if you like! I think something minimal would be the best.
I'll get working on the edits to be made in the guide for now then.

**Comment by shiffman** [comment]
Sounds great! The reason I am slow to do this one and #628 is changing the screenshots maybe requires doing all of them so that they have the same look and feel?

**Comment by DenisovichDev** [comment]
Yes I have that in my mind. I can try make it look similar with a bit of editing. And if it doesn't look very good at the end, we can keep it in hold while we get the screenshots ready for it. There's no rush anyway

**Comment by dipamsen** [comment]
For PR Templates, I think the following versions are possible:
- Porting a Track
- Porting a Challenge
- Showcase Submission

**Comment by dipamsen** [comment]
That is true!

**Comment by DenisovichDev** [comment]
Ah thank you @dipamsen!

**Comment by shiffman** [comment]
Is this still a valid issue? It's been 2 years now so I'm not sure if it's still relevant given where we are with the repo?

**Comment by shiffman** [comment]
I think I'll go ahead and close this old issue, we can always revisit updating the passenger showcase guide if there is a need.
